package.json

```diff
@@ -3009,6 +3009,15 @@
 						"default": false,
 						"description": "Whether to register the Dart SDK's MCP server with VS Code. This only applies to Dart SDKs >= v3.9.0 which added the server.",
 						"scope": "window"
+					},
+					"dart.mcpServerLogFile": {
```

Comment:
We should add this new setting in the "Logging" section rather than the "SDK" section.